### PR TITLE
Email Verification Resend

### PR DESCRIPTION
**Pull request by wyattjoh**
## What does this PR do?

Prompts the user to resend their email verification if it has not been received.

Related: https://www.pivotaltracker.com/story/show/151851092

### CODE REVIEW

File: services/limit.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  key(value) {
+    return `limit[${this.prefix}][${value}]`;
+  }
+
+  async get(value) {
+    const key = this.key(value);
+
+    return client().get(key);
+  }
+
```
Comment (alinhle) [inline services/limit.js:38]: Some comments for this method would be awesome :-)

**Comment by kgardnr** [comment]
Nice! This works well:

new_pass

My only question is I noticed 2 users being generated - shouldn't these be generated in the test db, not the dev db? @cvle @wyattjoh

**Review by kgardnr — CHANGES_REQUESTED** [review]
Outstanding question

**Review by kgardnr — APPROVED** [review]

**Comment by kgardnr** [comment]
I think the above is something with my local env, approving! 💯

**Pull request merged**